components/Card.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+
+const Card: React.FC<Props> = ({ btncolor, heading, paragraph, onClick }) => {
+  return (
+    <div
+      style={{
+        border: "2px solid black",
+        width: "250px",
+        height: "260px",
+        boxShadow: "#837d7d 0px 0px 10px 0px",
+      }}
+    >
+      <button
+        style={{
+          backgroundColor: btncolor,
+          marginTop: "20px",
+          marginLeft: "20px",
+          width: "50px",
+          cursor: "pointer",
+        }}
+      >
+        News
+      </button>
```

Comment:
If we have created button component then we need to reuse it here

Reply:
Updated